docker-compose.yml

```diff
@@ -1,29 +1,44 @@
-version: "3"
+version: '3'
 
 services:
   server:
     image: node:12.9.1
     command: >
-      yarn install
-      yarn dev
+      /bin/sh -c '
+        yarn --prefer-offline &&
+        yarn workspace @scorekeep/server dev'
     working_dir: /server
     volumes:
-      - ./packages/server:/server
+      - yarn-cache:/usr/local/share/.cache/yarn/v1
+      - $PWD:/server
     ports:
       - 3000:3000
     links:
-      - ui
+      - postgres
   ui:
     image: node:12.9.1
     command: >
-      yarn install
-      yarn dev
+      /bin/sh -c 'yarn workspace @scorekeep/ui dev'
     environment:
       REACT_APP_API_URL: http://server:3000
-    working_dir: /server
+    working_dir: /ui
     volumes:
-      - ./packages/ui:/server
+      - $PWD/.:/ui
+      - yarn-cache:/usr/local/share/.cache/yarn/v1
     ports:
       - 3100:3100
     links:
       - server
+  postgres:
+    image: postgres:11
+    restart: always
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: 'scorekeep-admin'
```

Comment:
In the near future move this to env variables

Reply:
we probably dont even need login credentials for a docker-compose database